hr_holidays_public/models/hr_leave.py

```diff
@@ -23,37 +23,22 @@ def action_validate(self):
                 leave = leave.with_context(
                     employee_id=leave.employee_id.id, exclude_public_holidays=True
                 )
-            super(HrLeave, leave).action_validate()
+            super(HrLeave, leave).action_validate(check_state)
```

Comment:
Always preserve keyword arguments:
```suggestion
            super(HrLeave, leave).action_validate(check_state=check_state)
```

Reply:
Thanks for your advice